src/gui/src/displayControls.cpp

```diff
@@ -351,6 +352,7 @@ DisplayControls::DisplayControls(QWidget* parent)
   makeLeafItem(misc_.instance_names, "Instance names", misc, Qt::Checked, false, instance_name_color_);
   makeLeafItem(misc_.scale_bar, "Scale bar", misc, Qt::Checked);
   makeLeafItem(misc_.fills, "Fills", misc, Qt::Unchecked);
+  makeLeafItem(misc_.access_points, "Access Points", misc, Qt::Checked);
```

Comment:
Does it make sense to have this on by default?

Reply:
No, I don't think. I'm gonna change it to unchecked.